src/clsourcedependencies_parser.cc

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
Do you work for google? If not, don't say you do.

Reply:
Removed.